sdk/tools/mkhive/rtl.c

```diff
@@ -138,15 +138,15 @@ RtlAssert(PVOID FailedAssertion,
 {
    if (NULL != Message)
    {
-      DbgPrint("Assertion \'%s\' failed at %s line %d: %s\n",
+      DbgPrint("Assertion \'%s\' failed at %s line %lu: %s\n",
```

Comment:
This is not correct in a host tool, ULONG is not `unsigned long`.

Reply:
Ah... Changed to `%u` then.